test/kuzzle/kuzzle.test.js

```diff
@@ -233,4 +237,48 @@ describe('/lib/kuzzle/kuzzle.js', () => {
       }
     });
   });
+
+  describe('#install', () => {
+    let handler;
+
+    beforeEach(() => {
+      handler = sinon.stub().resolves();
+      sinon.stub(Date, 'now').returns(Date.now());
+
+      kuzzle.ask = sinon.stub().withArgs(['core:storage:private:document:exist']).resolves(false);
+      kuzzle.ask = sinon.stub().withArgs(['core:storage:private:document:create']).resolves();
+    });
+
+    afterEach(() => {
+      Date.now.restore();
+    });
+
+    it('should call the handler and work properly', async () => {
+      const result = await kuzzle.install('id', handler);
+
+      should(kuzzle.ask).be.calledTwice();
```

Comment:
Can you check that `kuzzle.ask` has been called with the correct arguments please ? This function could literally be used to do anything :sweat_smile: